recipes/hangar/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 1
+  skip: true  # [py<36]
+  entry_points:
+    - hangar = hangar.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+
+requirements:
+  host:
+    - blosc-hdf5-plugin  # [not win]
+    - click
+    - grpcio
+    - h5py ==2.9.0
+    - msgpack-python ==0.6.1
+    - numpy
+    - pip
+    - protobuf
+    - python
+    - python-blosc
+    - python-lmdb >=0.92
+    - pyyaml
+    - tqdm
+    - wrapt
+  run:
+    - blosc-hdf5-plugin  # [not win]
```

Comment:
I guess this is just because `blosc-hdf5-plugin` isn't built for Windows? Would it be practical to make a pull request [here](https://github.com/conda-forge/blosc-hdf5-plugin-feedstock) so this can be `noarch: python`?

It's fine if you don't want to for whatever reason.

Reply:
So the `blosc-hdf5-plugin` library isn't a hard dependency for this project. It's just an optional package which mildly increases IO performance of compressed data.

It's actually a pretty huge achievement to make the `hdf5 blosc` filter available for Unix-like platforms alone. The `hdf5` plugin build/distribution/install system is fairly convoluted, and I'm not actively versed in its workings right now. It would take more time than I have right now to hack a window build script together. It might move up on my priority list in the future, but right now It's not feasible.

Since it's a totally optional package, i'm not worried about it being a major issue for distributing `Hangar` via `conda-forge`. 

Hopefully this is ok?

Reply:
Yeah that's fine, once @lantiga confirms I'll merge 👍